src/main/cpp/chassis/HolonomicDrive.cpp

```diff
@@ -315,8 +315,12 @@ void HolonomicDrive::InitSpeeds(double forwardScale,
     auto maxSpeed = m_swerve->GetMaxSpeed();
     auto maxAngSpeed = m_swerve->GetMaxAngularSpeed();
     auto scale = (FMSData::GetInstance()->GetAllianceColor() == frc::DriverStation::Alliance::kBlue) ? 1.0 : -1.0;
-    m_moveInfo.chassisSpeeds.vx = forwardScale * maxSpeed * scale;
-    m_moveInfo.chassisSpeeds.vy = strafeScale * maxSpeed * scale;
+
```

Comment:
This shouldn't be done here

This should be done similar to how other axis functionality is done (see gamepad/axis/AnalogAxis with how we apply deadbands, profile, scaling, inversion, etc.   Slew limiting should be done there.

Reply:
I agree with all of the scaling, deadbands, profiling, and inversion should be done in the telopcontrol/gamepad stuff. However, for slews limiters using units specifically shouldn't we do it in holonmic? I can move it, I don't have a problem, but I think for this specifically since we can control it through units, it may be easier to control (at least understand and tune the slews) in holonomic.cpp

Let me know what you think and I can update it if you still would like me to

Reply:
Maybe it is one of those things that we do it in both, so we have the ability to slew the axis's directly as well

Reply:
I wouldn't do it in both places.

Reply:
> I agree with all of the scaling, deadbands, profiling, and inversion should be done in the telopcontrol/gamepad stuff. However, for slews limiters using units specifically shouldn't we do it in holonmic? I can move it, I don't have a problem, but I think for this specifically since we can control it through units, it may be easier to control (at least understand and tune the slews) in holonomic.cpp
> 
> 
> 
> Let me know what you think and I can update it if you still would like me to  

Why can't the units be passed in?

Reply:
> I agree with all of the scaling, deadbands, profiling, and inversion should be done in the telopcontrol/gamepad stuff. However, for slews limiters using units specifically shouldn't we do it in holonmic? I can move it, I don't have a problem, but I think for this specifically since we can control it through units, it may be easier to control (at least understand and tune the slews) in holonomic.cpp
> 
> 
> 
> Let me know what you think and I can update it if you still would like me to  

Why can't the units be passed in?